lib/origen/utility/file_diff.rb

```diff
@@ -0,0 +1,214 @@
+module Origen
+  module Utility
+    module FileDiff
+      # Diff Processor (Origen::Utility::Processor) provides an easy way to diff the contents of two files
```

Comment:
Can you move this comment down to be above the Processor class, otherwise it will be lost and won't appear in the API